### PR TITLE
Remove premium_price attribute from registrar order responses

### PR DESCRIPTION
**Pull request by weppos**
See https://github.com/dnsimple/dnsimple-developer/pull/155

### CODE REVIEW

File: dnsimple/registrar.go [hunk 1]
```diff
@@ -122,6 +121,9 @@ type DomainRegisterRequest struct {
 	// Set to true to enable the auto-renewal of the domain.
 	// Default to true.
 	EnableAutoRenewal bool `json:"auto_renew,omitempty"`
+	// Required as confirmation of the price, only if the domain is premium.
+	// Default to true.
```
Comment (aeden) [inline dnsimple/registrar.go]: I don't understand why this says "Default to true" when this is a string value.

File: dnsimple/registrar.go [hunk 2]
```diff
@@ -175,6 +176,9 @@ type DomainTransferRequest struct {
...
```
Comment (aeden) [inline dnsimple/registrar.go]: I don't understand why this says "Default to true" when this is a string value.

File: dnsimple/registrar.go [hunk 3]
```diff
@@ -239,6 +242,9 @@ type domainRenewalResponse struct {
 type DomainRenewRequest struct {
 	// The number of years
 	Period int `json:"period"`
+	// Required as confirmation of the price, only if the domain is premium.
+	// Default to true.
```
Comment (aeden) [inline dnsimple/registrar.go]: I don't understand why this says "Default to true" when this is a string value.

File: dnsimple/registrar.go (hunk 1 above)
Comment (weppos) [reply dnsimple/registrar.go]: It is a mistake.

**Comment by weppos** [comment]
I fixed it @aeden

**Pull request merged**